sql/2024/fonts/design/fonts_foundry.sql

```diff
@@ -0,0 +1,48 @@
+-- Section: Design
+-- Question: Which foundries are popular?
+
+-- INCLUDE ../common.sql
+
+WITH
+foundries AS (
+  SELECT
+    client,
+    FOUNDRY(payload) AS foundry,
+    COUNT(DISTINCT page) AS count,
+    ROW_NUMBER() OVER (PARTITION BY client ORDER BY COUNT(DISTINCT page) DESC) AS rank
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-07-01' AND
+    type = 'font'
+  GROUP BY
+    client,
+    foundry
+  QUALIFY
+    rank <= 100
+),
+pages AS (
+  SELECT
+    client,
+    COUNT(DISTINCT page) AS total
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-07-01'
```

Comment:
Are we using the '2024-07-01' or '2024-06-01'?

Reply:
The former for fonts. We did some crawler changes after the latter.